entrypoint.sh

```diff
@@ -2,15 +2,26 @@
 
 set -eu
 
+API_HOST=0.0.0.0
+API_PORT=${API_PORT:-5000}
+API_WORKERS=${API_WORKERS:-3}
+API_WORKER_MAX_REQUESTS=${API_WORKER_MAX_REQUESTS:-10000}
 CELERY_WORKER_CONCURRENCY=${CELERY_WORKER_CONCURRENCY:-2}
 
 case "$1" in
+    api) poetry run gunicorn immuni_analytics.sanic:sanic_app \
+            --access-logfile='-' \
+            --bind=${API_HOST}:${API_PORT} \
+            --max-requests=${API_WORKER_MAX_REQUESTS} \
+            --workers=${API_WORKERS} \
+            --worker-class=uvicorn.workers.UvicornWorker ;;
     beat) poetry run celery beat \
             --app=immuni_analytics.celery.celery_app \
             --loglevel=debug ;;
     worker) poetry run celery worker \
             --app=immuni_analytics.celery.celery_app \
```

Comment:
So now we could add a `CELERY_WORKER_APP_NAME` to be valued something like `celery_app_<purpose_1>` and `celery_app<purpose_2>`?

```python
# old
--app=immuni_analytics.celery.celery_app \
# new
--app=immuni_analytics.celery.${CELERY_WORKER_APP_NAME} \
```